lib/internal/net.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Is a new module file necessary? This seems small enough that it could possibly just go into internal/util